modules/light-clients/08-wasm/internal/ibcwasm/wasm.go

```diff
@@ -1,14 +1,21 @@
 package ibcwasm
 
 import (
-	storetypes "cosmossdk.io/store/types"
+	"cosmossdk.io/collections"
+	storetypes "cosmossdk.io/core/store"
 )
 
 var (
 	vm WasmEngine
-	// storeKeyMap stores the storeKey for the 08-wasm module. Using a global storeKey is required since
-	// the client state interface functions do not have access to the keeper.
-	wasmStoreKey storetypes.StoreKey
+
+	// state management
+	Schema     collections.Schema
+	CodeHashes collections.KeySet[[]byte]
+
+	// keys:
```

Comment:
is this comment necessary? theres only one so far...

Reply:
removed